llama/patches/0022-metal-add-op_neg.patch

```diff
@@ -0,0 +1,75 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Michael Yang <git@mxy.ng>
+Date: Wed, 2 Apr 2025 15:26:15 -0700
+Subject: [PATCH] metal: add op_neg
```

Comment:
AFAIK we usually do `ggml_scale(ctx, cur, -1)` instead of using `ggml_neg`. The performance and memory should be the same with `ggml_scale_inplace`. Could this allow you to carry one less patch?